packages/medusa/src/repositories/product-category.ts

```diff
@@ -160,4 +158,40 @@ export const ProductCategoryRepository = dataSource
     },
   })
 
-export default ProductCategoryRepository
\ No newline at end of file
+export default ProductCategoryRepository
+
+const scopeChildren = (
+  category: ProductCategory,
+  treeScope: QuerySelector<ProductCategory> = {}
+): ProductCategory => {
+  if (isEmpty(treeScope)) {
```

Comment:
nit: I think we should try to avoid depending on lodash for those utils and create our own like the other isObject, isString etc. You decide what to do here :)

Reply:
Because I don’t see a reason to use a lib if we can do it easily natively.  

return isObject(obj) && !!Object.keys(obj).length)